Evaluation/Input/Content/Section3_Results_and_Discussion.md

```
@@ -0,0 +1,21 @@
+
+The model was built and evaluated covering data from studies including in particular:
+
+* single-dose and repeated-dose oral administration,
+* oral solution, capsule, and tablet formulations,
+* CYP2D6 extensive, normal, intermediate, and poor metabolizer settings represented by genotype, phenotype, or activity score.
+
+The model quantifies CYP2D6-mediated metabolism, CYP2C19-mediated non-CYP2D6 metabolism, and passive glomerular filtration.
+
+The next sections show:
+
+1. the final model input parameters for the building blocks: [Section 3.1](#31-atomoxetine-final-input-parameters).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)